Source/Particles/MultiParticleContainer.H

```diff
@@ -55,6 +55,9 @@ class MultiParticleContainer
 
 public:
 
+    using AllContainerType =
+        amrex::Vector<std::unique_ptr<WarpXParticleContainer>>;
```

Comment:
It seems that this is used only once. Is the goal to use it in more places? Otherwise, could we simply use `amrex::Vector<std::unique_ptr<WarpXParticleContainer>>`?

Reply:
You're right, the initial goal was to use it multiple times but it turns out I didn't need to so I'll just remove it.